packages/data-service-generator/src/server/resource/create-resource.ts

```diff
@@ -63,6 +64,19 @@ async function createResourceModules(entity: Entity): Promise<ModuleMap> {
 
   const [controllerModule, controllerBaseModule] = controllerModules.modules();
 
+  const controllerGrpcModules =
+    (appInfo.generateGrpc &&
```

Comment:
plus appInfo doesn't contain the new `mandatory` property `generateGrpc` in `@amplication/code-gen-types` v2.0.13

https://github.com/amplication/amplication/pull/7333/files#diff-0f8ca4854ebc0d04f3a5b0c2e2b33952568240b576a78fb49abce26afb688f31L8

Reply:
While we will remove this from the appInfo, Mor will anyway need to update the DSG' package.json to depend on 2.0.14 - you are right about that.